auth/tests/int/attestation.test.ts

```diff
@@ -1,18 +1,19 @@
 import { beforeAll, describe, expect, it } from "vitest";
 import { LoggingDriver } from "../driver/logging.driver"
 import { LambdaClient, InvokeCommand, LogType } from "@aws-sdk/client-lambda";
-import { fromSSO } from "@aws-sdk/credential-providers";
 import { testConfig } from "../common/config";
 const event = require("../fixtures/authProxyEvent.json")
 
 describe("attestation lambda", () => {
     const client = new LambdaClient({
         region: 'eu-west-2',
-        credentials: fromSSO()
```

Comment:
Set a:

```
[default]
region = 
output = json
sso_session = 
sso_account_id = 
sso_role_name = 
```

default profile in `/.aws/config` to be able to run vitest + debugging inline in vscode test explorer